Sources/NIOHTTP1Server/main.swift

```diff
@@ -527,7 +527,7 @@ let fileIO = NonBlockingFileIO(threadPool: threadPool)
 let socketBootstrap = ServerBootstrap(group: group)
     // Specify backlog and enable SO_REUSEADDR for the server itself
     .serverChannelOption(ChannelOptions.backlog, value: 256)
-    .serverChannelOption(ChannelOptions.socketOption(.reuseaddr), value: 1)
+    .serverOptions([.reuseAddr])
```

Comment:
Based on this example, the shorthand method doesn't save a lot unless we add a lot more shorthand definitions.

Reply:
I think `backlog` doesn't get used much in most cases.

I also think we're proposing adding these for `childChannelOption` on the `ServerBootstrap`, as well as for `ChannelOption` on the `ClientBootstrap`, `DatagramBootstrap`, and `PipeBootstrap`, which should provide substantially more help.

Reply:
I'll try adding a few more later to see how we generalise.

Reply:
I think we should actually add backlog. The default is super low so if I think real-world stuff sets it to something very high like 2048 to be sure connections aren't dropped if there's a quick burst.

Reply:
I think we decided not to add backlog in separate conversion.

Reply:
We really need backlog, it's something every real-world server has to set. When did we decide not to add?

Reply:
I lie - I've already done backlog.